app/uk/gov/hmrc/pillar2submissionapi/controllers/Pillar2ErrorHandler.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.pillar2submissionapi.controllers
+
+import play.api.http.HttpErrorHandler
+import play.api.libs.json.Json
+import play.api.mvc.{RequestHeader, Result, Results}
+import uk.gov.hmrc.pillar2submissionapi.controllers.error._
+
+import scala.concurrent.Future
+
+class Pillar2ErrorHandler extends HttpErrorHandler {
+
+  override def onClientError(request: RequestHeader, statusCode: Int, message: String): Future[Result] = ???
```

Comment:
We'll need to implement this, or just fall back to the client error on HttpErrorHandler

Reply:
yes, we do